plugin-server/src/main/ingestion-queues/session-recording/process-event.ts

```diff
@@ -332,6 +334,8 @@ export const createSessionReplayEvent = (
         event_count: Math.trunc(events.length),
         message_count: 1,
         snapshot_source: snapshot_source || 'web',
+        // we can't default this one, since we now have multiple libraries in production
+        snapshot_library: snapshot_library?.trim().length ? snapshot_library : null,
```

Comment:
any reason to trim? since we send the values ourselves here

Reply:
have learned not to trust the data at this point in the pipeline
(which is why we have all the `Math.trunc` and similar in this step

so, only to avoid the risk of an empty string but then we don't do that above so need to decide which is wrong 🤣